plutus-ledger-api/test-plugin/Spec/Data/Budget.hs

```diff
@@ -30,6 +31,8 @@ tests =
   runTestNested ["test-plugin", "Spec", "Data", "Budget"] . pure . testNestedGhc $
       [ goldenPirReadable "gt" compiledGt
       , goldenPirReadable "currencySymbolValueOf" compiledCurrencySymbolValueOf
+      , goldenEvalCekCatchBudget "mintValueMinted" compiledMintValueMinted
```

Comment:
There's no point doing a budget test on a function (`MintValue.MintValue -> Value`). A function cannot be evaluated further without an argument, so the evaluator doesn't do anything. Apply it to an argument to produce a `CompiledCode Value`, then pass it to `goldenEvalCekCatchBudget`. You can do so with `unsafeApplyCode`. See `DataList.Budget.Spec` for examples.